spec/models/registration_spec.rb

```diff
@@ -23,24 +23,42 @@
     Registration.new(registration_attrs)
   }
 
-  describe "address_label" do
-    it "saves Location name based on address_label" do
-      label = "The Address Label"
-      registration_attrs[:address_label] = label
+  describe '#save' do
+    describe "address_label" do
+      it "saves Location name based on address_label" do
+        label = "The Address Label"
+        registration_attrs[:address_label] = label
 
-      success = registration.save
-      expect(success).to be true
+        success = registration.save
+        expect(success).to be true
 
-      expect(registration.organization.locations.first.name).to eq(label)
+        expect(registration.organization.locations.first.name).to eq(label)
+      end
+
+      it "defaults to 'Default Address' when saving a new Location" do
+        registration_attrs[:address_label] = nil
+
+        success = registration.save
+        expect(success).to be true
+
+        expect(registration.organization.locations.first.name).to eq("Default Address")
+      end
     end
 
-    it "defaults to 'Default Address' when saving a new Location" do
-      registration_attrs[:address_label] = nil
+    describe 'user' do
+      it 'belongs to proper organization' do
+        success = registration.save
+        expect(success).to be true
+
+        expect(registration.user.organizations).to include(registration.organization)
+      end
 
-      success = registration.save
-      expect(success).to be true
+      it 'password is valid' do
```

Comment:
wording is a bit weird here, but no biggie

Reply:
My specs can be a bit awkward/verbose, especially when the specs start verifying nested data. In this case I'd write 'it should have a valid password'. 
if you're really bored one night, the generally expected style guide is at http://www.betterspecs.org/